src/components/SingleArticle/SingleArticle.tsx

```diff
@@ -1,15 +1,21 @@
 import React from 'react'
 import { DocumentRenderer } from '@keystone-6/document-renderer'
-
+import { Tag } from '@trussworks/react-uswds'
 import styles from './SingleArticle.module.scss'
-
 import type { ArticleRecord } from 'types'
+import colors from 'styles/sfds/colors.module.scss'
 
 export const SingleArticle = ({ article }: { article: ArticleRecord }) => {
   const {
     title,
     publishedDate,
     body: { document },
+    // TODO: Find a way to destructure this.
+    // I think I need to use nullish coalescing here,
+    // but can't figure out how to write it properly
+    // byline: { name: byline },
+    // location: { name: location },
```

Comment:
I made byline and location optional in the type, but can't figure out how to destructure them properly. I tried a few things: a ternary, default params, nullish coalescing, but none of that worked. Would love some input on this, but if we can't get it, then it works just fine declaring the old way as you can see below on lines 52, 55, 59 and 62.

Reply:
Guess the other option is to not make them optional, but that feels unnecessary since they are optional fields on article.